www/docs/js/main.js

```diff
@@ -349,4 +349,23 @@ $(function(){
     }
   });
 
+  var supportsDetailsElement = function(){
+    var el = document.createElement('details');
+    if ( ! ('open' in el) ) {
+      return false;
+    }
+    $('body')[0].appendChild(el);
```

Comment:
I mean, fine, but I think this was to work around a bug in Chrome 10? ;)

Reply:
Hah, you’re right. Just the `'open' in el` check is enough for IE9–11, which is what we care about here. :+1: